src/components/StyledTextarea.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class="StyledTextarea" @input="updateInput(true)" contenteditable></div>
```

Comment:
Class name in kebab-case
```suggestion
  <div class="styled-textarea" @input="updateInput(true)" contenteditable></div>
```